### PR TITLE
Entity Settings pages have explanation headers

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/i18n/en/translation.en.json [hunk 1]
```diff
@@ -1089,5 +1089,13 @@
   "associates-view": {
     "less": "show less",
     "more": "...plus {{count}} more"
+  },
```
Comment (techsmyth) [inline src/i18n/en/translation.en.json:1092]: I think that for the text to have meaningful content then it has to be specific to each tab that is shown. So I think there will need to be a translation section for each entity.tab. 

So there would need to be a hubs, users, organizations, authorization section underneath the admin level etc., with tabs below each of those. Make sense?

**Review by techsmyth — CHANGES_REQUESTED** [review]
Looks like a clean update. Nice to see it getting with less code  :)

Main feedback is that I think for these labels to have meaning then we will need to have them specific to each tab.

The organization tabs were for me not working properly after this update. Profile tab name was Context, and other tabs gave a 404:
image

File: src/i18n/en/translation.en.json (hunk 1 above)
Comment (me-andre) [reply src/i18n/en/translation.en.json:1092]: I updated the structure.
So now, whenever one comes up with a custom explanation for a tab under an entity, he just copies the part of the structure from "generic" under e.g. "hub" and edits it there:
```
    "hub": {
      "profile": { "explanation": "This is a very specific message about the Hub Profile." }
      ...
    }

```
Does it work for you, @techsmyth ?

**Comment by me-andre** [comment]
The bug with navigation under Organization is addressed.

File: src/i18n/en/translation.en.json (hunk 1 above)
Comment (techsmyth) [reply src/i18n/en/translation.en.json:1092]: Yes, thanks - this works...

**Review by techsmyth — APPROVED** [review]
Working nicely.

**Pull request merged**